EKF/ekf_helper.cpp

```diff
@@ -852,72 +861,172 @@ void Ekf::calcEarthRateNED(Vector3f &omega, float lat_rad) const
 	omega(2) = -CONSTANTS_EARTH_SPIN_RATE * sinf(lat_rad);
 }
 
-// gets the innovations of velocity and position measurements
-// 0-2 vel, 3-5 pos
-void Ekf::get_vel_pos_innov(float vel_pos_innov[6])
+void Ekf::getGpsVelPosInnov(float hvel[2], float &vvel, float hpos[2],  float &vpos)
+{
+	memcpy(hvel, _gps_vel_pos_innov+0, sizeof(float) * 2);
+	memcpy(&vvel, _gps_vel_pos_innov+2, sizeof(float) * 1);
+	memcpy(hpos, _gps_vel_pos_innov+3, sizeof(float) * 2);
+	memcpy(&vpos, _gps_vel_pos_innov+5, sizeof(float) * 1);
+}
+
+void Ekf::getGpsVelPosInnovVar(float hvel[2], float &vvel, float hpos[2], float &vpos)
+{
+	memcpy(hvel, _gps_vel_pos_innov_var+0, sizeof(float) * 2);
+	memcpy(&vvel, _gps_vel_pos_innov_var+2, sizeof(float) * 1);
+	memcpy(hpos, _gps_vel_pos_innov_var+3, sizeof(float) * 2);
+	memcpy(&vpos, _gps_vel_pos_innov_var+5, sizeof(float) * 1);
+}
+
+void Ekf::getGpsVelPosInnovRatio(float &hvel, float &vvel, float &hpos, float &vpos)
+{
+	memcpy(&hvel, _gps_vel_pos_test_ratio+HVEL, sizeof(float));
```

Comment:
These ones with the references are much easer to read as:
```suggestion
	hvel = _gps_vel_pos_test_ratio[HVEL];
```